tools/picard.in

```diff
@@ -15,6 +15,10 @@ if [ "$RUN_DEBUG" != "" ]; then
     DEBUG_COMMAND="gdb --args"
 fi
 
+if [ "$RUN_PROFILE" != "" ]; then
+    export EOS_PROFILE=1
+fi
```

Comment:
Picard doesn't use EknContent nor make_ready probes (uses a dummy engine).

Reply:
It's entirely possible that the probes are not tripped directly; I'm going through the code to check.